src/clang.rs

```diff
@@ -2030,9 +2030,6 @@ pub struct TargetInfo {
 impl TargetInfo {
     /// Tries to obtain target information from libclang.
     pub fn new(tu: &TranslationUnit) -> Option<Self> {
```

Comment:
This can just return `Self` now, and the fallback code on the caller removed, which is a nice simplification.

Reply:
Yes, there are a lot of such transformations that can be done, and since you seem to be fine with the idea, I will start collecting them in this PR.

Reply:
(Unless you would prefer those changes to be in a separate PR.)

Reply:
Upon a closer look, I see that I never would have found this particular case, so thanks for pointing it out.